### PR TITLE
Add Abilities for EnragedSpirits Script

### PR DESCRIPTION
**Pull request by AnonXS**

### CODE REVIEW

File: src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp [hunk 1]
```diff
@@ -176,7 +176,7 @@ CreatureAI* GetAI_mob_azaloth(Creature *_creature)
 
 enum 
 {
-    SPELL_SHADOWBOLT = 9613,
+    SPELL_SHADOWBOLT_9613 = 9613,
```
Comment (Xadras) [inline src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp]: Is there any reason for doing this? ^^

File: src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp [hunk 2]
```diff
@@ -1872,54 +1872,175 @@ CreatureAI* GetAI_mob_torloth_the_magnificent(Creature* c)
...
+            {
+                if (StormboltTimer <= diff)
+                {
+                    DoCast(m_creature->getVictim(), SPELL_STORMBOLT);
+                    StormboltTimer = urand(3400, 4700);
+                }
+                else
+                    StormboltTimer -= diff;
+            break;
+            }}}}
```
Comment (Xadras) [inline src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp]: What the kek :D
This should be solved in another way ;D

File: src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp (hunk 1 above)
Comment (AnonXS) [inline src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp]: vs complained about it, but i guess its no problem to name two Spells SPELL_SHADOWBOLT in the same .cpp file if they are defined at the right place befor they are used? so does he take the new defined SPELL_SHADOWBOLT for each script that follows after that?

File: src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp (hunk 2 above)
Comment (AnonXS) [inline src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp]: yea should have looked up a similar case for how it's done properly.

File: src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp (hunk 1 above)
Comment (Xadras) [inline src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp]: yep, but in this case that solution here is alright, improves readability
-> didn't check for more SPELL_SHADOWBOLT

**Pull request merged**